src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -0,0 +1,828 @@
+        onSelectParticipant?: (option: Participant) => void;
+
+        /** Should we request a single or multiple participant selection from user */
+        hasMultipleParticipants: boolean;
+
+        /** IOU amount */
+        iouAmount: number;
+
+        /** IOU comment */
+        iouComment?: string;
+
+        /** IOU currency */
+        iouCurrencyCode?: string;
+
+        /** IOU type */
+        iouType?: ValueOf<typeof CONST.IOU.TYPE>;
+
+        /** IOU date */
+        iouCreated?: string;
+
+        /** IOU merchant */
+        iouMerchant?: string;
+
+        /** IOU Category */
+        iouCategory?: string;
+
+        /** IOU Tag */
+        iouTag?: string;
+
+        /** IOU isBillable */
+        iouIsBillable?: boolean;
+
+        /** Callback to toggle the billable state */
+        onToggleBillable?: () => void;
+
+        /** Selected participants from MoneyRequestModal with login / accountID */
+        selectedParticipants: Participant[];
+
+        /** Payee of the money request with login */
+        payeePersonalDetails?: OnyxEntry<OnyxTypes.PersonalDetails>;
+
+        /** Can the participants be modified or not */
+        canModifyParticipants?: boolean;
+
+        /** Should the list be read only, and not editable? */
+        isReadOnly?: boolean;
+
+        /** Depending on expense report or personal IOU report, respective bank account route */
+        bankAccountRoute?: string;
+
+        /** The policyID of the request */
+        policyID?: string;
+
+        /** The reportID of the request */
+        reportID?: string;
+
+        /** File path of the receipt */
+        receiptPath?: string;
+
+        /** File name of the receipt */
+        receiptFilename?: string;
+
+        /** List styles for OptionsSelector */
+        listStyles?: StyleProp<ViewStyle>;
+
+        /** ID of the transaction that represents the money request */
+        transactionID?: string;
+
+        /** Whether the money request is a distance request */
+        isDistanceRequest?: boolean;
+
+        /** Whether the money request is a scan request */
+        isScanRequest?: boolean;
+
+        /** Whether we're editing a split bill */
+        isEditingSplitBill?: boolean;
+
+        /** Whether we should show the amount, date, and merchant fields. */
+        shouldShowSmartScanFields?: boolean;
+
+        /** A flag for verifying that the current report is a sub-report of a workspace chat */
+        isPolicyExpenseChat?: boolean;
+
+        hasSmartScanFailed?: boolean;
+
+        reportActionID?: string;
+
+        transaction: OnyxTypes.Transaction;
+    };
+
+function MoneyRequestConfirmationList({
+    onConfirm = () => {},
+    onSendMoney = () => {},
+    onSelectParticipant = () => {},
+    iouType = CONST.IOU.TYPE.REQUEST,
+    iouCategory = '',
+    iouTag = '',
+    iouIsBillable = false,
+    onToggleBillable = () => {},
+    payeePersonalDetails = null,
```

Comment:
I think we can skip it
```suggestion
    payeePersonalDetails,
```